woocommerce/woocommerce
# Add/legacy assets build

### All Submissions:






### Changes proposed in this Pull Request:



Closes #31132

This is a continuation work based on 13-gh-woocommerce/woocommerce-monorepo

It adds an implicit dependency package for legacy CSS and JS files.

### How to test the changes in this Pull Request:

1. Run `pnpm install`.
2. Run `pnpm nx build woocommerce-legacy-assets`.
3. Ensure assets are generated into `plugins/woocommerce/assets` directory.
4. Delete all files from `plugins/woocommerce/assets/js` and `plugins/woocommerce/assets/css`.
5. Run `pnpm nx build woocommerce`.
6. Ensure assets are generated into `plugins/woocommerce/assets` directory.

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### FOR PR REVIEWER ONLY:
